randovania/games/super_metroid/logic_database/Maridia.txt

```diff
@@ -960,8 +1063,15 @@ Extra - asset_id: 158
   * Super Missile Door to The Precious Room/Door to Colosseum
   > Door to Colosseum Save Room
       Any of the following:
-          Grapple Beam
-          Gravity Suit and Space Jump
+          All of the following:
+              # With Gravity
+              Gravity Suit
+              Any of the following:
+                  Space Jump or Wall Jumps (Intermediate)
+                  # IBJ
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
+          # Without Gravity
+          Grapple Beam and Wall Jumps (Intermediate)
```

Comment:
do we count the WJ off of grapple blocks really as a walljump? feels kinda wrong.

Reply:
I was unsure, but it is a wall jump.  Would you prefer there be no walljumps for these?

Reply:
can you send a vid? if it is what i think i is, i'd prefer no WJ.

Reply:
https://youtu.be/tIn8EO5KRss

Reply:
yeah would say no WJ.

Reply:
removed wall jumps